frontend/__tests__/containers/MetricsStep/PipelineConfiguration/PipelineConfiguration.test.tsx

```diff
@@ -153,7 +149,7 @@ describe('DeploymentFrequencySettings', () => {
 
     await userEvent.click(getByTestId('AddIcon'));
 
-    expect(addADeploymentFrequencySetting).toHaveBeenCalledTimes(1);
+    expect(addAPipelineSetting).toHaveBeenCalledTimes(1);
```

Comment:
PLZ assert the parameters of the `addAPipelineSetting`

Reply:
this function is no any parameter